httplib.h

```diff
@@ -786,6 +787,7 @@ class Server {
   Server &set_file_extension_and_mimetype_mapping(const std::string &ext,
                                                   const std::string &mime);
   Server &set_default_file_mimetype(const std::string &mime);
+  Server &set_default_content_mimetype(const std::string &mime);
```

Comment:
Could you remove `set_default_content_mimetype` please? You can see the default with `set_default_headers`.